### PR TITLE
Updates to FMI import documentation

### PR DESCRIPTION
**Pull request by casella**
This PR updates the User's Guide documentation of the FMI import feature, according to the latest discussions.

It should be merged once and only if all issues in #5345 are resolved.

### CODE REVIEW

**Comment by AnHeuermann** [comment]
@casella This PR conflicts with https://github.com/OpenModelica/OpenModelica/pull/9455. Should I just remove my changes to the FMI Import section?

**Comment by adeas31** [comment]
> @casella This PR conflicts with #9455. Should I just remove my changes to the FMI Import section?

Yes. We decided in the last dev meeting to fix the FMI import. See also #9525 and #9532

File: doc/UsersGuide/source/fmitlm.rst [hunk 1]
```diff
@@ -91,50 +91,53 @@ SUNDIALS manualy and create a build script as well.
...
+and only represents continuous time dynamic behaviour, it should work without problems
+when imported as a Modelica block.
+
+Please also note that the current implementation of FMI Import in OpenModelica
+is based on a built-in wrapper that uses a `reinit()` statement in an algorithm
+section. This is not allowed by the Modelica Language Specification, so it is
+necessary to set the compiler to accept this non-standard construct by setting
+the `--allowNonStandardModelica=reinitInAlgorithms` compiler flag. In OMEdit,
+you can set this option by activating the *Enable FMU Import* checkbox in the
+*Simulation Setup | Translation Flags* tab. This will generate a warning during
```
Comment (adeas31) [inline doc/UsersGuide/source/fmitlm.rst:119]: Its better to write,

"In OMEdit, you can set this option by activating the *Enable FMU Import* checkbox in the
*Tools | Options | Simulation | Translation Flags*."

Of course one can overwrite the option in `Simulation Setup`.

**Review by adeas31 — CHANGES_REQUESTED** [review]

File: doc/UsersGuide/source/fmitlm.rst (hunk 1 above)
Comment (casella) [reply doc/UsersGuide/source/fmitlm.rst:119]: OK

**Pull request merged**